lib/master.js

```diff
@@ -361,17 +369,6 @@ class Master extends EventEmitter {
         from: 'app',
       });
     });
-    cluster.on('listening', (worker, address) => {
-      this.messenger.send({
-        action: 'app-start',
-        data: {
-          workerPid: worker.process.pid,
-          address,
-        },
-        to: 'master',
-        from: 'app',
-      });
-    });
```

Comment:
Why remove this one?

Reply:
send port to master after egg ready and before server listen.